test/scheduler/tainttoleration_integration_test.go

```diff
@@ -0,0 +1,702 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+package tests
+
+import (
+	"fmt"
+	"strconv"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/ptr"
+
+	clusterv1beta1 "go.goms.io/fleet/apis/cluster/v1beta1"
+	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+)
+
+var _ = Describe("scheduling CRPs on member clusters with taints & tolerations", func() {
```

Comment:
add a test, create crp, add a taint to the existing cluster (selected by the crp), to validate if the placement will be removed from the cluster or not